src/tools/compiletest/tidy-rustdoc.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+indir="${1:?Missing argument 1: input directory}"
+
+tidy () {
+  {
+    # new-inline-tags is workaround for:
+    #   https://github.com/rust-lang/stdarch/issues/945
+    #   https://github.com/rust-lang/mdBook/issues/1372
+    command tidy \
+        --indent yes \
+        --indent-spaces 2 \
+        --wrap 0 \
+        --show-warnings no \
+        --markup yes \
+        --quiet yes \
+        --new-inline-tags 'c t' \
+        "$@" \
+        >/dev/null \
+    || {
+      # tidy exits with code 1 if there were any warnings :facepalm:
+      status=$?
+      if [ $status != 0 ] && [ $status != 1 ]
+      then
+        echo "While tidying $1" >&2
+        exit 1
+      fi
+    }
+  } | sed -E 's/#[0-9]+(-[0-9]+)?/#line/g'
```

Comment:
You don't need the line-number removal for this usecase, I needed it to be able to compare modified `std` without a bunch of noise from line numbers.